controller/api/serializers.py

```diff
@@ -239,23 +239,22 @@ def validate_domain(self, value):
         """
         Check that the hostname is valid
         """
-        match = re.match(
-            r'^(\*\.)?(' + settings.APP_URL_REGEX + r'\.)*([a-z0-9-]+)\.([a-z0-9]{2,})$',
-            value)
-        if not match:
+        if len(value) > 255:
```

Comment:
note: we can use these validators if we shift this over to the model, which ay be a better idea (I did this in https://github.com/deis/deis/pull/2898)